### ISSUE TITLE
Buffer.concat incorrectly flagged as prefer-spread

### ISSUE THREAD

**Issue by marklawlor** [sindresorhus/eslint-plugin-unicorn#1068]
The NodeJS Buffer class has a static `.concat` function that is being flagged as a violation of `prefer-spread - Array#concat()`

Example usage (taken from https://nodejs.org/api/buffer.html#buffer_static_method_buffer_concat_list_totallength)

```
// Create a single `Buffer` from a list of three `Buffer` instances.

const buf1 = Buffer.alloc(10);
const buf2 = Buffer.alloc(14);
const buf3 = Buffer.alloc(18);
const totalLength = buf1.length + buf2.length + buf3.length;

console.log(totalLength);
// Prints: 42

const bufA = Buffer.concat([buf1, buf2, buf3], totalLength);

console.log(bufA);
// Prints: <Buffer 00 00 00 00 ...>
console.log(bufA.length);
// Prints: 42
```

The Buffer class should be ignored by the `prefer-spread` rule.

**Comment by fisker** [comment]
I'll fix.
